python/src/lib.rs

```diff
@@ -60,6 +104,28 @@ impl RawDeltaTable {
     pub fn file_paths(&self) -> PyResult<Vec<String>> {
         Ok(self._table.get_file_paths())
     }
+
+    pub fn schema(&self, format: &str) -> PyResult<DeltaTableSchema> {
+        let schema = match &format[..] {
+            "ARROW" => {
+                <ArrowSchema as From<&deltalake::Schema>>::from(self._table.schema().unwrap())
```

Comment:
we should propagate the error through `PyResult` instead of calling unwraps. In general, `unwraps` and `panic` should be avoided.